examples/LatentNO/LatentNO-time.py

```diff
@@ -0,0 +1,191 @@
+import hydra
+import paddle
+from omegaconf import DictConfig
+from utils import RelLpLoss_time
+
+import ppsci
+
+
+def train(cfg: DictConfig):
+
+    model = ppsci.arch.LatentNO_time(**cfg.MODEL)
+
+    train_dataset_cfg = {
+        "name": "LatentNODataset_time",
+        "data_name": cfg.data_name,
+        "data_mode": "train",
+        "data_normalize": cfg.data_normalize,
+        "data_concat": cfg.data_concat,
+        "input_keys": ("x", "y1", "y2"),
+        "label_keys": ("y2",),
+    }
+    train_ds = ppsci.data.dataset.build_dataset(train_dataset_cfg)
+
+    if cfg.data_normalize:
+        normalizer = train_ds.normalizer
+    else:
+        normalizer = None
+
+    train_dataloader_cfg = {
+        "dataset": {
+            "name": "LatentNODataset_time",
+            "data_name": cfg.data_name,
+            "data_mode": "train",
+            "data_normalize": cfg.data_normalize,
+            "data_concat": cfg.data_concat,
+            "input_keys": ("x", "y1", "y2"),
+            "label_keys": ("y2",),
+        },
+        "sampler": {"name": "BatchSampler", "drop_last": True, "shuffle": True},
+        "batch_size": cfg.TRAIN.train_batch_size,
+        "num_workers": cfg.get("num_workers", 0),
+    }
+
+    eval_dataloader_cfg = {
+        "dataset": {
+            "name": "LatentNODataset_time",
+            "data_name": cfg.data_name,
+            "data_mode": "val",
+            "data_normalize": cfg.data_normalize,
+            "data_concat": cfg.data_concat,
+            "input_keys": ("x", "y1", "y2"),
+            "label_keys": ("y2",),
+        },
+        "sampler": {"name": "BatchSampler", "drop_last": True, "shuffle": False},
+        "batch_size": cfg.EVAL.eval_batch_size,
+        "num_workers": cfg.get("num_workers", 0),
+    }
+
+    train_loss_fn = RelLpLoss_time(
+        p=2, key="y2", normalizer=None, use_full_sequence=False
+    )
+    val_loss_fn = RelLpLoss_time(
+        p=2, key="y2", normalizer=normalizer, use_full_sequence=True
+    )
+
+    sup_constraint = ppsci.constraint.SupervisedConstraint(
+        train_dataloader_cfg,
+        train_loss_fn,
+        output_expr={"y2": lambda out: out["y2"]},
+        name="SupTrain",
+    )
+
+    constraint = {sup_constraint.name: sup_constraint}
+    lr_scheduler = ppsci.optimizer.lr_scheduler.OneCycleLR(
+        epochs=cfg.TRAIN.epochs,
+        iters_per_epoch=cfg.iters_per_epoch,
+        max_learning_rate=cfg.lr,
+        divide_factor=cfg.div_factor,
+        end_learning_rate=cfg.lr / cfg.div_factor / cfg.final_div_factor,
+        phase_pct=cfg.pct_start,
+    )()
```

Comment:
抱歉我此前没关注到你用LR是需要`iters_per_epoch`的，可以这样改一下，其余几个案例文件也作同样修改
```suggestion
    cfg.TRAIN.iters_per_epoch = len(sup_constraint.data_loader)
    lr_scheduler = ppsci.optimizer.lr_scheduler.OneCycleLR(
        epochs=cfg.TRAIN.epochs,
        iters_per_epoch=cfg.TRAIN.iters_per_epoch,
        max_learning_rate=cfg.lr,
        divide_factor=cfg.div_factor,
        end_learning_rate=cfg.lr / cfg.div_factor / cfg.final_div_factor,
        phase_pct=cfg.pct_start,
    )()
```